test/nightwatch/book-class.js

```diff
@@ -37,13 +37,14 @@ module.exports = {
       .waitForElementVisible('body.home', 1000)
       .url(config.web.url + '/classes')
       .waitForElementVisible('body.classes', 1000)
-      .click('#date option[value="' + moment(cls.startTimes[0]).format('YYYY-MM') + '"]')
+      .click('#month option[value="' + moment(cls.startTimes[0]).format('YYYY-MM') + '"]')
       .waitForElementVisible('body.classes', 1000)
 
     browser = clickByText('.sow-panel span', classTemplate.name, browser)
       .waitForElementVisible('body.class', 1000)
       .click('.btn.book-now')
       .waitForElementVisible('body.basket', 1000)
+      .pause(250)
```

Comment:
Same again, is it possible to wait for `#basket tr:first-child .product-description` to be visible?